### ISSUE TITLE
Vimeo video not loading: Franz Wanner

### ISSUE THREAD

**Issue by xcvy** [ns-dokuzentrum-muenchen/departure-neuaubing-frontend#146]
Issue Vimeo video not loading: The english version of Franz Wanner's page had the same issue as the german page. Refresh video and saving it resolved the issue. 

Was this maybe something specific that happened with an update to the page?

If it happens for another reason, can we avoid it somehow or will we need to manually check the videos regularly?

**Comment by n-kort** [comment]
@xcvy I'm pretty sure the access token stored in the ACF field went stale, but I don't know when or why this would happen. Could actually be after settings changed on the Vimeo side too.

**Comment by xcvy** [comment]
Yes that seems logical but I asked and they claim there where no changes to the video made. It is weird because it happened to both the German and the English page. I will try to keep an eye out if it happens to more videos.
